dist/component-war/src/test/java/org/hawkular/bus/PublishDataPointsTest.java

```diff
@@ -74,6 +75,8 @@
 import rx.Observable;
 
 /**
+ * TODO: We need to mock out AlertsService, I think, so that InsertedDataSubscriber can instantiate correctly
```

Comment:
You can add a Custom implementation of AlertsService on the test folder and add it manually as a class.